### ISSUE TITLE
Re-enable LLAMA

### ISSUE THREAD

**Issue by dctcd** [Antyk45/IBM-Chatbot#75]
LLAMA has been disabled for better accessibility during the demo. This was in an effort to speed up our demo in the short allocated timeframe. This should be re-enabled.

**Comment by nassir90** [comment]
No longer needed as a task as LLMs can be enabled / disabled at run time.

**Comment by nassir90** [comment]
Feature was addressed in #78
